cellpack/bin/simularium_converter.py

```diff
@@ -201,17 +238,19 @@ def get_ingredient_data(self, cytoplasm, main_container, ingredient):
         return (ingredient_name, cytoplasm_data, container_data)
 
     def get_euler_from_matrix(self, data_in):
-        rotation_matrix = np.array(
-            [
-                np.array(data_in[0][0:3]),
-                np.array(data_in[1][0:3]),
-                np.array(data_in[2][0:3]),
-            ]
-        ).transpose()
-        return R.from_matrix(rotation_matrix).as_euler("xyz", degrees=True)
+        rotation_matrix = [
+            np.array(data_in[0][0:3]),
+            np.array(data_in[1][0:3]),
+            data_in[2][0:3],
+        ]
+        return np.degrees(
+            tr.euler_from_matrix(rotation_matrix)
+        )  # R.from_matrix(rotation_matrix).as_euler("xyz", degrees=True)
```

Comment:
this may be a good idea, but I tried this and it still seems like the rotations are incorrect

Reply:
the euler angle calculated here are the correct one, and show up correctly in Blender. Do you know the coordinate system used in simularium ? Y is up ? Z is forward or backward ?

Reply:
<img width="660" alt="Screen Shot 2021-11-15 at 6 08 19 PM" src="https://user-images.githubusercontent.com/5170636/141882891-890500c3-8554-49d8-932b-c72fd66ea662.png">

Reply:
so the camera is in the positive Z direction in this view

Reply:
this is definitely an improvement since they all seem off in the same way

Reply:
the problem is either in the 3d geometry export/import or in how the rotation is applied. Because the euler passed on this  model should be all (0,0,z) since its a 2d packing.

Reply:
Ok, I got it to look like the picture you posted in blender, but I had to hard code the x and z rotations. One thing I realized is that simularium is actually in radians. I think it was once in degrees, but either way that took longer than it should have for me to realize. I think simularium is also importing the geometry incorrectly: if I send in 0,0,0 for the rotation, the meshes are all standing up in Z, so I had to first rotate them 90 degrees to get them to lay flat. I also haven't figure out why the bounding box is off for this recipe, i haven't seen that in other recipes I've converted and looking at the positions, it seems like they aren't shifted or anything.

Reply:
I dig up a little more that particular recipe, and realize it was setup in left hand coordinate system and the collada files were coming from Cinema4D which is left hand. The ingredient actually is defined in the lefthand coordinate system for the mesh. When I exported the file in obj and then reloaded them in blender, it automatically rotate 90degree on X. So it was both the .obj and the viewer ! This recipe should be deprecated, and I will try to prepare a cleaner one.

Reply:
awesome, yeah, thanks for digging in.... I just pushed a commit that has it using radians but not messing with any of the coordinates systems